### PR TITLE
[install README] update install doc, including make - major

### PR DESCRIPTION
**Pull request by kongtiaowang**
on the `major` branch: Add this [Run make (for production instances) or make dev (for development sandboxes). ] into readme for Ubuntu installs 
CentOS related update: #5202 

to help solve issue #5197 
also related to #5198 : install script messaging update 

Similar changes will be issued to `bugfix` for 21 release which is out already

### CODE REVIEW

**Comment by christinerogers** [comment]
Since this is on the major branch, the version number should say *22*

File: README.md [hunk 1]
```diff
@@ -66,39 +66,42 @@ Consult the [LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) page on this
...
-    _Note_: Apache config files will be installed as *.conf, per Ubuntu 14.04. If running an earlier version of Ubuntu, rename these files, then run the following commands. After, restart Apache.
+5. Apache configuration
 
+If your apache configuration was not completed by the Install script, run the following enable rewriting of LORIS, enable your `$projectname` site, and restart apache:
 
     ```
+    sudo a2enmod rewrite
     sudo a2dissite default
     sudo a2ensite $projectname
+    sudo service apache2 reload
```
Comment (PapillonMcGill) [inline README.md]:
```suggestion
    sudo systemctl reload apache2
```
New way with systemd,  old way still work

Comment (kongtiaowang) [reply README.md]: @PapillonMcGill 
In the other Readme files and wikis, We always use the "sudo service apache2 reload". If you prefer the new way, we could change it in the other PR to change all of them.

Comment (PapillonMcGill) [reply README.md]: Yes, we can do it in an separate PR, to have the same every where.

**Comment by christinerogers** [comment]
@PapillonMcGill I don't see why we shouldn't make the apache reload change (from your resolved comment) in the currently open PRs -- . 
To confirm -- is it actually better to use  (a) `sudo systemctl reload apache2`  over (b) `sudo service apache2 reload` ? 
If so, let's do it (consistently).

**Comment by PapillonMcGill** [comment]
@christinerogers both way work, using the old way (system apache2 reload) will be automaticaly translated to the new way. 
As per @kongtiaowang recommendation, another PR will be created to replace the call in all documentation at the same time.

**Comment by driusan** [comment]
This is going to the wrong branch.. the instructions apply to all branches now that the release is out, not just major.

**Comment by kongtiaowang** [comment]
@christinerogers @PapillonMcGill 
I will close this PR, it is on the wrong branch as @davidblader said.

File: README.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ Deploy and log in with username *admin* and the password that's set up during de
  * Apache **2.4** or higher
  * MySQL >= 5.7 (or MariaDB >= 10.3) 
  * PHP <b>7.2</b> or higher
+ * Node >= 6.17
  * [Composer](https://getcomposer.org/)
 
 _Composer should be run with --no-dev option unless you are an active LORIS developer._
```
Comment (christinerogers) [inline README.md]: @kongtiaowang  can we remove this line?    `make` or `make dev` should cover this, right?

**Pull request closed without merging**